ivy/functional/ivy/experimental/layers.py

```diff
@@ -2480,3 +2480,81 @@ def ifftn(
             [-0.48472244+0.30233797j]])
     """
     return ivy.current_backend(x).ifftn(x, s=s, axes=axes, norm=norm, out=out)
+
+
+@handle_nestable
+def gru(
+    x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+    w: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+    r: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+    b: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+    /,
+    *,
+    initial_h: Optional[Union[ivy.Array, ivy.NativeArray, ivy.Container]] = None,
+) -> Tuple[ivy.Array, ivy.Array]:
+    """
+    Compute an one-layer GRU. This operator is usually supported via some custom
```

Comment:
This documentation you took it from ONNX documentation you need to write the documentation in your own according to paramters and special cases